pkg/istio/rbac/common/util.go

```diff
@@ -96,6 +98,24 @@ func (a *ApiHandler) Delete(item *Item) error {
 	return err
 }
 
+// CheckIfMemberIsAllUsers returns namespace for Athenz domain when role member is of form '<athenz-domain>.*'.
+// Example: domain.sub-domain.* -> domain-sub--domain
+func CheckIfMemberIsAllUsers(member *zms.RoleMember, domainName zms.DomainName) (string, error) {
+
+	if member == nil {
+		return "", fmt.Errorf("member is nil")
+	}
+
+	memberStr := string(member.MemberName)
```

Comment:
there is no need to do string cast since MemberName is already type String

Reply:
`MemberName` is of type `zms.MemberName` and needs to be converted to a string before proceeding.